scripts/apps/archive/constants.ts

```diff
@@ -59,6 +59,16 @@ export enum ITEM_STATE {
      */
     RECALLED = 'recalled',
 
+    /**
+     * Correction, If Correction workflow is true, Correction is send on a desk.
+     */
+    CORRECTED = 'corrected',
+
```

Comment:
Can you rephrase comments so they are more clear? Now I don't understand any additional information from them that is not already in the enum name.

I assume if an item is `CORRECTED` it will always published and items that are `BEING_CORRECTED`, will always be not published, but that were published in the past?